### PR TITLE
Changed the location of google maps

### PR DESCRIPTION
**Pull request by Manan-dev**
I moved it to where Andre wanted and I linked a screenshot below of the mobile view.

Screenshot 2021-08-09 171546

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -250,11 +250,15 @@ <h3>∞</h3>
             <div class="col-lg-8 col-lg-offset-2">
                 <h2>The Venue</h2>
                 <p>
-                    The Min Kao building is one of the newest buildings on campus, with an expansive interior, modern
+                    The <a href="https://www.eecs.utk.edu/overview/facilities/" target="_blank">Min H. Kao Engineering building</a> is one of the newest buildings on campus, with an expansive interior, modern
                     architecture, and ridiculous internet speeds. Min Kao houses the Department of Electrical
-                    Engineering and Computer Science, and is a hotspot of innovation at the university. This weekend, it
+                    Engineering and Computer Science, and is a hotspot of innovation at the university. During VolHacks, it
```
Comment (andreicozma1) [inline index.html:255]: Cleaning up diff here as it's not related to this issue

**Comment by andreicozma1** [comment]
What does it look like on desktop in full screen view? Perhaps we could have the maps be a little less tall (both movile and desktop) to take less screen space. However still enough such that it's easy to navigate.
Lmk what you guys think @Manan-dev @hunterprice04

**Comment by Manan-dev** [comment]
Screenshot 2021-08-10 153644
This is the desktop view

**Comment by Manan-dev** [comment]
This is the improved version of mobile view as per your recommendations
Screenshot 2021-08-10 155935

**Comment by andreicozma1** [comment]
The desktop view looks really good! One thing for the mobile view, is there any way to have the mobile view wider (as wide as the text) while also keeping desktop view the same (or close to) what it is now?
For mobile I think it's not wide enough to be able to easily navigate it. The height is good though
@Manan-dev

**Comment by Manan-dev** [comment]
image

yeah, I just adjusted the width of the mobile view. Lmk how it looks

**Comment by andreicozma1** [comment]
> image
> 
> 
> 
> yeah, I just adjusted the width of the mobile view. Lmk how it looks

So much better!

**Review by andreicozma1 — APPROVED** [review]

**Pull request merged**